nlubridge/nlu_dataset.py

```diff
@@ -31,227 +30,258 @@ class EntityKeys:
     TYPE = "entity"
     START = "start"
     END = "end"
+    CONFIDENCE = "confidence"
+
+
+class NBestKeys:
+    """Keys for n-best list items (dicts) in NLUdataset."""
+
+    INTENT = "intent"
+    CONFIDENCE = "confidence"
 
 
 class NluDataset:
     def __init__(
         self,
         texts: List[str],
         intents: Optional[List[str]] = None,
-        entities: Optional[List[List[dict]]] = None,
-        out_of_scope=False,
+        entities: Optional[
+            List[List[Dict]]
+        ] = None,  # TODO: Better code entities as custom objects?
+        n_best_intents: Optional[
+            List[List[Dict]]
+        ] = None,  # TODO: Better code n-best-list items as custom objects?
         max_intent_length=None,
-        seed=42,
-    ):
+    ) -> None:
         """
         Class for managing NLU data with intent and entity annotations.
 
         :param texts: A list of utterances
-        :type texts: List[str]
         :param intents: A list of intents for each text. If intents is not
             empty then there needs to be an intent for each text.
-        :type intents: List[str]
         :param entities: If intents is not empty then there needs to be an
             intent for each text.
-        :type entities: List[dict]
-        :param max_intent_length: If given the intents will be strimmed to
-            the first `max_intent_length` in chars. This is required
+        :param n_best_intents: List of dicts with predicted intent and confidence
```

Comment:
Vielleicht besser 
"List of n-best results (each denoted as a dict having intent and confidence keys) for all samples"
schreiben, da es ja List[List[Dict]] ist.

Reply:
hab ich so angepasst und bei der Gelegenheit auch die anderen Beschreibungen gefixt.